src/ModelerApp.vue

```diff
@@ -61,6 +63,11 @@ export default {
   },
   data() {
     return {
+      self: this,
+      validationBar: [],
+      decorations: {
+        borderOutline: {},
+      },
```

Comment:
It seems like `decorations` only contains `borderOutline`, in which case, why wrap that property? Can `borderOutline` be referenced and passed down directly?

Reply:
Well the idea for use "decorations" is to group any other styling decorations of the diagram (outside of the diagram bpmn definitions). I know at the moment this PR only provides borderOutline, but I think in the future, other decorations could be required, like:

- change task colors (in runtime)
- opacity (to show elements that were disabled or failed)
- add icons (to show that some tasks have uploaded files or comments)